devices/r4igoldcc.cpp

```diff
@@ -115,32 +117,39 @@ class R4SDHC_DualCore : Flashcart {
         state.key2.seed_byte = 0;
         m_card->setBlowfishState(platform::getBlowfishKey(key), key != BlowfishKey::NTR);
         if ((err = m_card->beginKey1())) {
-            logMessage(LOG_ERR, "R4SDHC: trySecureInit: init key1 (key = %d) failed: %d", static_cast<int>(key), err.errNo());
+            logMessage(LOG_ERR, "R4iGoldCC: trySecureInit: init key1 (key = %d) failed: %d", static_cast<int>(key), err.errNo());
             return false;
         }
         if ((err = m_card->beginKey2())) {
-            logMessage(LOG_ERR, "R4SDHC: trySecureInit: init key2 failed: %d", err.errNo());
+            logMessage(LOG_ERR, "R4iGoldCC: trySecureInit: init key2 failed: %d", err.errNo());
             return false;
         }
 
         return true;
     }
 
 public:
-    R4SDHC_DualCore() : Flashcart("R4 SDHC Dual Core", 0x200000) { }
+    R4iGoldCC() : Flashcart("R4i Gold CC", 0x200000) { }
 
     const char* getAuthor() {
         return
                     "handsomematt, Normmatt, Kitlith,\n"
             "        stuckpixel, angelsl, et al.";
     }
+    const char *getDescription() {
+        return "Works with many various R4iGoldCC type flashcarts\n" 
+               " * R4i Gold CC (r4igold.cc)\n"
+               " * R4 SDHC Dual-Core (r4isdhc.hk)\n"
+               " * R4iTT 3DS (r4itt.net)\n"
+               " * R4i XDS 2014 (r4ixds.com)";
```

Comment:
This (r4ixds) actually doesn't appear to be the same cart, on a second look, so please remove it for now :)